.zenodo.json

```diff
@@ -0,0 +1,43 @@
+{
+    "creators": [
+        {
+            "orcid": "0000-0001-8313-1644",
+            "name": "Chapoton, Frédéric"
+        },
+        {
+            "name": "Davis, Diana"
+        },
+        {
+            "orcid": "0000-0002-9608-782X",
+            "name": "Delecroix, Vincent",
+            "affiliation": "CNRS - Université de Bordeaux"
+        },
+        {
+            "name": "Fontaine, Oscar"
+        },
+        {
+            "name": "Fougeron, Charles"
+        },
+        {
+            "name": "Jeffreys, Luke"
+        },
+        {
+            "orcid": "0000-0001-8313-1644",
+            "name": "Lelièvre, Samuel",
+            "affiliation": "Université Paris-Saclay"
+        },
+        {
+            "orcid": "0000-0002-3930-9107",
+            "name": "Rüth, Julian"
+        },
+        {
+            "name": "Yakovlev, Ivan"
+        },
+        {
+            "name": "Zhang, Christopher"
+        }
+    ],
```

Comment:
@videlec @slel if you happen to know ORCIds or affiliations, feel free to fill them in here.